src/main/java/org/traccar/protocol/FreematicsProtocolDecoder.java

```diff
@@ -146,8 +149,23 @@ private Object decodePosition(
                 case 0x82:
                     position.set(Position.KEY_DEVICE_TEMP, Integer.parseInt(value) * 0.1);
                     break;
+                case 0x104:
+                    position.set(Position.KEY_ENGINE_LOAD, Integer.parseInt(value));
+                    break;
+                case 0x105:
+                    position.set(Position.KEY_COOLANT_TEMP, Integer.parseInt(value));
+                    break;
+                case 0x10c:
+                    position.set(Position.KEY_RPM, Integer.parseInt(value));
+                    break;
+                case 0x10d:
+                    position.set(Position.KEY_OBD_SPEED, UnitsConverter.knotsFromKph(Integer.parseInt(value)));
+                    break;
+                case 0x111:
+                    position.set(Position.KEY_THROTTLE, Integer.parseInt(value));
+                    break;
                 default:
-                    position.set(data[0], value);
+                    position.set(Position.PREFIX_IO + data[0], value);
```

Comment:
Looks like you forgot to replace `data[0]` with `key`.

Reply:
I left it as the hex representation because I felt like OBD PIDs are more often referred to by their hex ID rather than the integer equivalent, but if having it as the integer is more consistent with the other decoders I'll change that as well.

Reply:
Yes, it's for consistency.